content/about.html

```diff
@@ -18,24 +18,23 @@
           </div>
           <div class="masthead-left">
             <h1 class="masthead-left__title no-link">
-              What is web.dev?
+              Where you can learn & measure
             </h1>
             <div class="masthead-left__body">
               <p>
-                web.dev is the ultimate resource for developers of all
-                backgrounds to learn, create, and solve on the web. It's meant
-                to not only educate developers, but help them apply what they've
-                learned to any site they work on, be it personal or business.
+                Whether you already have a website or you're just getting started, you 
+                can learn to build for the modern web at web.dev. Then, apply your new 
+                skills and solutions to any personal or professional site you work on.
               </p>
               <p>
-                web.dev was borne of a belief that if we make high quality
-                experiences easy to build, it will enable more meaningful
-                engagement on the web—for users and developers alike. Simply
-                put, we realized the only way the web gets better is if we help
-                the people building it succeed.
+                After you've launched your site, come back to web.dev and measure how 
+                well it supports your users. If there are areas where it can improve, 
+                you'll get immediate steps to increase your metrics.
               </p>
               <p>
-                And the web can be better.
+                As the web becomes more complex, users' expectations get higher. With 
```

Comment:
Taking Harleen's comment on board, what about "As the web advances, users' expectations get higher"?